java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S888_java.json

```diff
@@ -18,5 +18,10 @@
   "defaultSeverity": "Critical",
   "ruleSpecification": "RSPEC-888",
   "sqKey": "S888",
-  "scope": "All"
+  "scope": "All",
+  "securityStandards": {
```

Comment:
@agigleux is it expected to have security standard for code smell?

Reply:
In that case, yes, because CWE is not only related to Security. It references weakness that can be bugs and I till tend to think RSPEC-888 should be a bug rather than a code smell.